### PR TITLE
Added obfuscation formatting feature. 

### PR DESCRIPTION
**Pull request by davidjpowell80**
Added a section to the README.md file which contains a link to a Wiki that is not published yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This Data Migration project provides a flexible powerful engine based on the XRM
 ## Table Of Contents
 1. [Installation](#Installation)
 1. [Usage](#Usage)
+1. [Obfuscation](#Obfuscation)
```
Comment (ewingjm) [inline README.md:12]: It seems like it would be better as a subheading under usage

**Comment by ewingjm** [comment]
Can't seem to comment on the file directly as it's too big. Is the full CSV of all UK postcodes necessary? It's 8.5MB. Seems like a sample may be easier to understand with just a few records.

File: samples/DemoScenarios/Contacts/Schema.xml [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-          <relationships />
-       </entity>
- </entities>
\ No newline at end of file
+<entities >
+	<entity name="contact" displayname="Contact" etc="2" primaryidfield="contactid" primarynamefield="fullname" disableplugins="false">
+		<fields>
+			<field displayname="First Name" name="firstname" type="string" />
+			<field displayname="Last Name" name="lastname" type="string" />
+			<field displayname="Parent Contact" name="parentcontactid" type="entityreference" lookupType="contact" />
```
Comment (ewingjm) [inline samples/DemoScenarios/Contacts/Schema.xml:6]: Was this change intended? Seems as though it's for a separate demo scenario.

File: src/Capgemini.Xrm.DataMigration.FileStore/DataStore/DataFileStoreWriter.cs [hunk 3]
```diff
@@ -98,6 +100,29 @@ public void SaveBatchDataToStore(List<EntityWrapper> entities)
...
+        /// <param name="entitiesToExport">Collection of entities being exported.</param>
+        public void RemoveEntityReferenceNameProperty(List<CrmEntityStore> entitiesToExport)
+        {
+            entitiesToExport.ThrowArgumentNullExceptionIfNull(nameof(entitiesToExport));
+
+            if (this.fieldsToObfuscate != null && this.fieldsToObfuscate.Any())
+            {
+                foreach (var entity in entitiesToExport)
+                {
+                    if (entity.Attributes.Where(x => x.AttributeType == "Microsoft.Xrm.Sdk.EntityReference").Any())
```
Comment (ewingjm) [inline src/Capgemini.Xrm.DataMigration.FileStore/DataStore/DataFileStoreWriter.cs:115]: Think the `if` statement here is redundant

File: tests/Capgemini.DataMigration.Core.Tests.Unit/Model.FieldToBeObfuscatedTests.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Capgemini.DataMigration.Core.Tests.Unit
+{
+    [TestClass]
```
Comment (ewingjm) [inline tests/Capgemini.DataMigration.Core.Tests.Unit/Model.FieldToBeObfuscatedTests.cs:11]: Empty test class

File: samples/DemoScenarios/Contacts/Schema.xml (hunk 2 above)
Comment (davidjpowell80) [reply samples/DemoScenarios/Contacts/Schema.xml:6]: This was just left over changes as I used the contact scenario when first developing the Obfuscation feature. I'll set it back to its original state.

File: README.md (hunk 1 above)
Comment (davidjpowell80) [reply README.md:12]: I'll make this change

File: src/Capgemini.Xrm.DataMigration.FileStore/DataStore/DataFileStoreWriter.cs (hunk 3 above)
Comment (davidjpowell80) [reply src/Capgemini.Xrm.DataMigration.FileStore/DataStore/DataFileStoreWriter.cs:115]: Agreed.
